templates/explore/pinned_repos_list.tmpl

```diff
@@ -0,0 +1,61 @@
+{{if (gt .PinnedReposNum 0)}}
+<h2>{{.i18n.Tr "explore.pinned_repos"}}</h2>
+<div class="ui cards">
+    {{range .PinnedRepos}}
+    <div class="card">
+      <div class="content">
+        <div class="header">
+            {{if .RelAvatarLink}}
+                <img class="ui avatar image" src="{{.RelAvatarLink}}">
+            {{end}}
+            <a class="name" href="{{.Link}}">
+                {{if $.Owner}}{{if (ne .Owner.ID $.Owner.ID)}}{{.Owner.Name}} / {{end}}{{end}}{{.Name}}
+			</a>
+			{{if $.CanConfigPinnedRepos}}
+				<button class="ui text grey pinned-repo-btn poping up" 
+					data-content="{{if .IsPinned}}{{$.i18n.Tr "settings.unpinned_repo"}}{{else}}{{$.i18n.Tr "settings.pinned_repo"}}{{end}}"
+					data-url="{{$.ConfigPinnedReposLink}}"
+					data-name="{{if $.Owner}}{{if (ne .Owner.ID $.Owner.ID)}}{{.Owner.Name}}/{{end}}{{end}}{{.Name}}"
+					data-status="{{if .IsPinned}}unpinned{{else}}pinned{{end}}">
+					{{if .IsPinned}}{{$.i18n.Tr "settings.unpinned_btn"}}{{else}}{{$.i18n.Tr "settings.pinned_btn"}}{{end}}
+				</button>
```

Comment:
You're going to need to provide the information to relabel this button once it's been clicked.